internal/utils/net.go

```diff
@@ -56,21 +51,8 @@ func ResolveHostIP(ctx context.Context, hostIP string) (subnets []string, privat
 	for _, address := range addrs {
 		private = private || isPrivateIP(address.IP)
 
-		// figure out the IP range
-		ipRange := ipv6FilterRange
```

Comment:
Removing this here is correct since we want to save IP addresses but that means we have IP addresses in the `ipFilter` now instead of subnets which means we need to move this to the `ipFilter` to avoid breaking it.